src/views/Amalgamation/Information.vue

```diff
@@ -71,7 +72,22 @@
         </template>
       </ExpandableHelp>
 
-      <ResultingBusinessName class="mt-5" />
+      <p
+        v-if="isAmalgamationFilingVertical"
+        class="mt-3">
+        This is a vertical short form amalgamation under section 273 of Business Corporations Act.
+        The amalgamated business will adopt as its notice of articles, the notice of articles of the holding business.
+      </p>
+      <p
+        v-if="isAmalgamationFilingHorizontal"
+        class="mt-3">
+        This is a horizontal short form amalgamation under section 273 of Business Corporations Act.
+        The amalgamated business will adopt as its notice of articles, the notice of articles of the primary business.
+      </p>
+      <ResultingBusinessName
+        :isRegularAmalgamation="isAmalgamationFilingRegular"
+        class="mt-5"
+      />
```

Comment:
What does this section look like for a short-form amalgamation?

In particular, I'm curious what draft/store variable sets the single Correct Name Option (which I think is CORRECT_AML_ADOPT)?

Reply:
This is on dev:
![image](https://github.com/bcgov/business-create-ui/assets/17242795/5bc92a22-7cfe-46da-820c-3ccd5279f8da)
This is on the PR:
![image](https://github.com/bcgov/business-create-ui/assets/17242795/1d564c24-bebe-4c7e-8664-f01e863bccd7)

Reply:
OK, your PR screenshot looks good, but where did the Correct Name Option get set? What do you see on a brand-new amalgamation draft filing?

Reply:
![image](https://github.com/bcgov/business-create-ui/assets/17242795/d6bcc324-eebb-4759-8a11-d4c634503b53)
You go through here first.  (No changes on the code were made here)

Reply:
I think we want this option automatically selected. Please check with Yui.

It should be easy to set in the code -- it's the `correctNameOption` field in the nameRequest object. I've been thinking about this and we might want to set this option (and the legalName) when we create the short-form amalgamation drafts -- in Filings UI. Look it over and see what you think.

Reply:
Yeah, because we removed the "Undo" button for short forms anyway. Will check with Yui about this.

Reply:
Resolved.